discovery-provider/alembic/trigger_sql/handle_repost.sql

```diff
@@ -105,6 +105,55 @@ begin
       on conflict do nothing;
     end if;
 
+	-- notify followees of the reposter who have reposted the same content
+	-- within the last month
+	if new.is_delete is false
+	and new.is_repost_repost is true then
+	with
+	    followee_repost_repost_ids as (
+	        select user_id
+	        from reposts r
+	        where
+	            r.repost_item_id = new.repost_item_id
+	            and new.created_at + INTERVAL '-1 month' < r.created_at
```

Comment:
could also do `new.created - INTERVAL '1 month' - I think it's a little bit more clear, bit either way works